lndclient/lightning_client.go

```diff
@@ -394,6 +397,24 @@ func (s *lightningClient) ListTransactions(ctx context.Context) ([]*wire.MsgTx,
 	return txs, nil
 }
 
+// ListChannels retrieves all channels of the backing lnd node.
+func (s *lightningClient) ListChannels(ctx context.Context) (
+	[]*lnrpc.Channel, error) {
```

Comment:
The idea of the wrapper is to not expose things like string based pubkeys. Ideally we'd create a strongly-typed struct to return a slice of. Can for now just contains the fields that we need.

Reply:
done